src/components/ItaliaTheme/View/PersonaView/PersonaDocumenti.jsx

```diff
@@ -205,6 +215,46 @@ const PersonaDocumenti = ({ content }) => {
           />
         </RichTextSection>
       )}
+      {content.dichiarazioni_di_insussistenza_e_incompatibilita?.download && (
+        <RichTextSection
+          tag_id="dichiarazioni_di_insussistenza_e_incompatibilita"
+          title={intl.formatMessage(
+            messages.dichiarazioni_di_insussistenza_e_incompatibilita,
+          )}
+        >
+          <div className="card-wrapper card-teaser-wrapper card-teaser-wrapper-equal">
+            <Attachment
+              download_url={
+                content.dichiarazioni_di_insussistenza_e_incompatibilita
+                  .download
+              }
+              title={
+                content.dichiarazioni_di_insussistenza_e_incompatibilita
+                  .filename
+              }
+            />
+          </div>
+        </RichTextSection>
+      )}
```

Comment:
Mi pareva esistesse un componente "Attachments" che si può usare al posto di fare RichTextSection + Attachment. Non ho verificato, ti chiedo se puoi dare un'occhiata per scrupolo intanto che torni su questa PR. Grazie

Reply:
Ho controllato. Attachments viene usato quando ci sono delle cartelle figlie in cui caricare i documenti. Viene fatta la query e vengono disegnate le card per fare il download in base al download_url creato a partire dai dati scaricati da questa query. 
Volendo si può modificare il componente per accomodare anche i file caricati direttamente dal form di edit del CT, oppure usare Attachment passando direttamente il download_url

Reply:
Va benissimo così come hai fatto allora, grazie! Mi ricordavo male io.